public/faq.md

```diff
@@ -1 +1,100 @@
-<!-- TODO: Write FAQs-->
+# GENERAL
+
+- __I do not have any prior knowledge of git and/or any programming language. Can I do this?__
+
+  - __Absolutely__. This program is specifically catered to the growth and smooth transition of people like you who want to learn. KOSS and the mentors will help you wherever possible.
+  - To learn more about git and github,check [these](https://github.com/kossiitkgp/Git-and-Github-2021).
+
+- __What is the deadline for selecting projects? How many projects can we select?__
+
+  - There is no deadline for selecting projects nor is there any restriction on the number of projects you can contribute to 
+  - you can also switch between projects as much as you want. The only necessity is that you should have atleast one merged PR per week.
+
+- __Where does KWoC occur? Do I need to be a student of IIT Kharagpur?__
+
+  Kharagpur Winter of Code is to be done remotely from home, and it is not required for you to be student of **ANY** particular institute.
+
+- __What is the purpose of this program? Is this some kind of internship?__
+  
+  **No**, KWoC is a program which aims at helping participants learn how to contribute in the open-source world. This activity prepares you for various open source opportunities such as [Google Summer of Code](https://summerofcode.withgoogle.com/archive/), [Rail Girl's Summer of Code and Outreachy](https://railsgirlssummerofcode.org).
+
+- __Can I participate in KWoC as both a mentor and a student?__
+
+  We do encourage participation into project development from everyone; but if you are a mentor we won't be publishing your stats nor will you be going through evaluations.
+
+- __What can I do to help spread the word about KWoC?__
+
+  Nothing makes us happier than people spreading words about open-source; to begin, you can like our [facebook page](https://www.facebook.com/kossiitkgp) and increase awareness about KWoC in your college; nothing works better than word of mouth.
+
```

Comment:
Add links to other koss platforms, twitter and linkedin.

Reply:
resolved